src/components/HypothesisBuilder/components/HypothesisNodeNew/index.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Disclosure } from "@headlessui/react";
+import { ChevronDownIcon } from "@heroicons/react/solid";
+import cx from "clsx";
+import { Icon } from "../../../Icon";
+import { HypothesisNodeItem } from "./HypothesisNodeItem";
+
+export const HypothesisAccordion = ({
+  node,
+  parentArray,
+  setModalIsOpen,
+  setSelectedNodePath
+}) => {
+  const handleOpenModal = () => {
+    setSelectedNodePath([...parentArray, node.id]);
+    setModalIsOpen(true);
+  };
+  const getStatusIcon = (status) => {
+    switch (status) {
+      case "proven":
+        return "like";
+      case "possible":
+        return "search";
+      case "disproven":
+        return "dislike";
+      default:
+        return "like";
+    }
+  };
+
+  return (
+    <div>
+      <div className="flex align-baseline text-base font-semibold mb-6">
+        <h2 className="text-dark-gray mr-3 font-inter text-2xl">Action plan</h2>
+        <div className="bg-dark-blue rounded-full p-2">
+          <Icon name="addButton" className="w-3 h-3" />
+        </div>
+      </div>
+      <div className="w-full mt-5">
+        <div className="w-full bg-white rounded-8px font-inter">
+          <Disclosure key={node.id}>
+            {({ open }) => (
+              <>
+                <Disclosure.Button className="w-full flex justify-between">
```

Comment:
The expand and collapse happens for content inside an individual hypothesis such as comments / evidence etc - not on the hypothesis themselves which are always shown (they would only be hidden when filtering e.g. exclude disproven hypothesis, which is onlhy to be added in a future phase)

Reply:
I tried a lot but I can't figure it out...

We are rendering accordions where header is hypothesis details row, and it's always visible, and content is evidences and solutions, which are not collapsible. 
Design:
![image](https://user-images.githubusercontent.com/20907048/152345547-eef75cbd-298c-4e96-a821-bd00d30b80a3.png)
Actual:
![image](https://user-images.githubusercontent.com/20907048/152345677-811c4988-8b3b-4aef-9a96-2e8313f206c7.png)
![image](https://user-images.githubusercontent.com/20907048/152345770-24f50cb4-2f8e-4d50-b2b5-ba47348460c2.png)

What's exactly wrong here ? Thank you

Reply:
There should't be any accordions in this initial phase it should be fully expanded, later in a nother phase we would bring more details from the Modal into a epandable section per hypothesis i.e. you can expand an issue / solution/ etc to see more details from the modal

Reply:
@moshloop 
![image](https://user-images.githubusercontent.com/20907048/153216094-3f835aff-3517-40cf-8240-431a2c8cedc9.png)

is the element outlined in the screenshot an incident with type "root", which is presented in the array of hypothesis, returned as part of "/db/incident" request?